content/lessons/table_of_contents.json

```diff
@@ -1,36 +1,53 @@
 {
   "id": "table-of-contents",
-  "coolness": "very-cool",
-  "chapters": {
-    "one": {
-      "subchapter_one_one": [
+  "chapters": [
+    {
+      "title": "The first chapter",
```

Comment:
Make the title a proper string and do not parse a JSON key to get the String to be displayed in the HTML. The approach chosen before (parsing the json key - not value - via [this function](https://github.com/ericwindmill/gatsby-starter-docs/blob/1967f125f938bb25b72ea00f6dcc5c5eec7d6520/src/components/Layout/TableOfContents.jsx#L13-L15)) is a quite bad idea. How would you ever want to account for chapter titles like `"The BigNum Problem"` or i18n?